sniproxy.cpp

```diff
@@ -157,23 +157,17 @@ SNIProxy::SNIProxy(xcb_window_t wid, QObject* parent):
                              windowMoveConfigVals);
 
 
-    //if the window is a clearly stupid size resize to be something sensible
-    //this is needed as chormium and such when resized just fill the icon with transparent space and only draw in the middle
-    //however spotify does need this as by default the window size is 900px wide.
-    //use an artbitrary heuristic to make sure icons are always sensible
-    if (clientGeom->width < 12 || clientGeom->width > s_embedSize ||
```

Comment:
Yes, the sizes are different but those sizes are original ones...
Ok, I'll try that smooth scaling on pixmap. Hope that would actually make a difference.